Neuron/Sections/Wallet/View/MnemonicViewController.swift

```diff
@@ -83,18 +74,8 @@ class MnemonicViewController: UITableViewController, ImportWalletViewModelDelega
         self.navigationController?.pushViewController(qrCtrl, animated: true)
     }
 
-    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        if indexPath.section == 1 && indexPath.row == 0 {
-            pickerView.frame = CGRect(x: 0, y: 0, width: ScreenSize.width, height: ScreenSize.height)
-            pickerView.delegate = self
-            pickerView.dataArray = [["name": "m/44'/60'/0'/0/0", "id": "0"], ["name": "m/44'/60'/0'/0", "id": "1"], ["name": "m/44'/60'/1'/0/0", "id": "2"]]
-            pickerView.selectDict = ["name": devirationPath.text!, "id": selectFormatId]
-            UIApplication.shared.keyWindow?.addSubview(pickerView)
-        }
-    }
-
     @IBAction func importWallet(_ sender: UIButton) {
-        viewModel.importWalletWithMnemonic(mnemonic: mnemonic!, password: password!, confirmPassword: confirmPassword!, devirationPath: devirationPath.text!, name: name!)
+        viewModel.importWalletWithMnemonic(mnemonic: mnemonic!, password: password!, confirmPassword: confirmPassword!, name: name!)
```

Comment:
Let's get rid of view models (could be from another future PR).

Reply:
OK